sdk/version.go

```diff
@@ -53,8 +51,20 @@ func GetVersion(dir string) (ver string, err error) {
 	return
 }
 
+func GetVersionFromDir(dir string) (string, error) {
+	return getVersion(dir, "COREOS_VERSION=")
+}
+
 func GetLatestVersion() (string, error) {
-	return GetVersion(BuildImageDir("latest"))
+	return getVersion(BuildImageDir("latest"), "COREOS_VERSION=")
+}
+
+func GetSDKVersion() (string, error) {
+	return getVersion(filepath.Join(RepoRoot(), "manifests"), "COREOS_SDK_VERSION=")
```

Comment:
Repo root points to the top level of the whole tree, so this should be `Join(RepoRoot(), ".repo", "manifests")`